tests/queries/0_stateless/02251_last_day_of_month.sql

```diff
@@ -44,3 +44,12 @@ SELECT toLastDayOfMonth(date_value), toLastDayOfMonth(date_time_value), toLastDa
 WITH
     toDate('2020-12-12') AS date_value
 SELECT last_day(date_value), LAST_DAY(date_value);
+
+-- boundaries
+WITH
+    toDate('1070-01-01') AS date_value,
+    toDateTime('1970-01-01 11:22:33') AS date_time_value,
+    toDateTime64('1900-01-01 11:22:33', 3) AS date_time_64_value
+SELECT toLastDayOfMonth(date_value), toLastDayOfMonth(date_time_value), toLastDayOfMonth(date_time_64_value)
+SETTINGS enable_extended_results_for_datetime_functions = true;
```

Comment:
Like to add a test for `toRelativeWeekNum()` at it's upper boundary?

EDIT: As far as I read, that test already existed (--> 01921_datatype_date32.reference).